src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/ECDsa/ECDsaTests.cs

```diff
@@ -517,6 +517,7 @@ public void Roundtrip_WithOffset(ECDsa ecdsa)
             Assert.True(ecdsa.VerifyData(halfData, dataSignature, HashAlgorithmName.SHA256));
         }
 
+#if netcoreapp // uses ECParameters not available in desktop.
```

Comment:
In most places haven't we been putting such tests in separate files rather than using ifdefs?